### PR TITLE
Fixing bat file

### PR DESCRIPTION
**Pull request by michaelhidalgo**
Fixing reload file.

### CODE REVIEW

File: bin/reload_data.bat [hunk 1]
```diff
@@ -15,6 +30,7 @@ echo "... deleting tm-uno-loaded.flag if exist..."
 IF EXIST %FileToDelete% del \F %FileToDelete%
 
 echo "... deleting cache files..."
-IF EXIST data_cache rmdir data_cache \s \q
-echo "... Data has been published"
-cd ..\..\..\bin\
\ No newline at end of file
+
+IF EXIST data_cache rd /s /q data_cache
```
Comment (DinisCruz) [inline bin/reload_data.bat]: shouldn't this be rmdir?

File: bin/reload_data.bat [hunk 2]
```diff
@@ -1,12 +1,27 @@
...
 cd ..\..\..\
 
-cd code\TM_GraphDB\.tmCache
+echo "...copy Graph_Data..."
+echo "%dir%"
+echo F| XCOPY .\data\Lib_UNO\__Graph_Data .\data\Lib_UNO-json\Graph_Data /i
+
+cd .\code\TM_GraphDB\.tmCache
 echo "%dir%"
 @REM File to be deleted
```
Comment (DinisCruz) [inline bin/reload_data.bat]: this single REM statement looks odd

Comment (michaelhidalgo) [inline bin/reload_data.bat]: thx, it has been removed.

**Comment by romichg** [comment]
Looks good. Merging (Sorry @DinisCruz, need this one in there)

**Pull request merged**